Testing/CICDTests/Services/FindDirServiceTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿// <copyright file="FindDirServiceTests.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+using System.IO.Abstractions;
+using CICDSystem.Services;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace CICDSystemTests.Services;
+
+/// <summary>
+/// Tests the <see cref="FindDirService"/> class.
+/// </summary>
+public class FindDirServiceTests
+{
+    private const string ProjectName = "MyProject";
+    private const string SolutionDir = $@"C:/{ProjectName}";
+    private const string DebugDir = $@"C:/{ProjectName}/{ProjectName}/bin/debug";
+    private const string BinDir = $@"C:/{ProjectName}/{ProjectName}/bin";
+    private const string ProjectDir = $@"C:/{ProjectName}/{ProjectName}";
+    private const string RootRepoDir = $@"C:/{ProjectName}";
+    private readonly Mock<IDirectory> mockDirectory;
+    private readonly Mock<IPath> mockPath;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FindDirServiceTests"/> class.
+    /// </summary>
+    public FindDirServiceTests()
+    {
+        this.mockDirectory = new Mock<IDirectory>();
+        this.mockDirectory.Setup(m => m.GetCurrentDirectory()).Returns(DebugDir);
+
+        // Starting at the execution dir
```

Comment:
I know this is a comment, but to be consistent, refactor dir to directory.  
```suggestion
        // Starting at the execution directory
```